coolwsd.xml.in

```diff
@@ -274,6 +274,10 @@
         <handling desc="How to handle fonts mising in a document: 'report', 'log', 'both', or 'ignore'" type="string" default="log">log</handling>
     </fonts_missing>
 
+    <indirection_endpoint>
+        <url desc="URL endpoint to collabora kubernetes operator which servers routeToken and serverId in json format" default=""></url>
```

Comment:
Can you replace all 'kubernetes operator' with 'indirection endpoint' here and at every place in the source code. The indirection doesn't have to be via an operator. Thanks !